src/libsync/vfs/cfapi/hydrationjob.cpp

```diff
@@ -118,13 +117,20 @@ void OCC::HydrationJob::start()
 
 void OCC::HydrationJob::cancel()
 {
+    _isCanceled = true;
+
     if (!_job) {
         return;
     }
 
     _job->cancel();
 }
 
+bool OCC::HydrationJob::hydrationCanceled() const
```

Comment:
@FlexW This can just be called `canceled` as `hydration` is already in the name of the class. Also, the getter name must conform to the `Qt` style. So, could be better for it to be named `isCanceled`.